src/main/java/io/github/nucleuspowered/nucleus/modules/note/listeners/NoteListener.java

```diff
@@ -0,0 +1,79 @@
+import io.github.nucleuspowered.nucleus.dataservices.loaders.UserDataManager;
+import io.github.nucleuspowered.nucleus.internal.ListenerBase;
+import io.github.nucleuspowered.nucleus.internal.PermissionRegistry;
+import io.github.nucleuspowered.nucleus.internal.permissions.PermissionInformation;
+import io.github.nucleuspowered.nucleus.internal.permissions.SuggestedLevel;
+import io.github.nucleuspowered.nucleus.modules.note.config.NoteConfigAdapter;
+import io.github.nucleuspowered.nucleus.modules.note.handlers.NoteHandler;
+import org.spongepowered.api.Sponge;
+import org.spongepowered.api.entity.living.player.Player;
+import org.spongepowered.api.event.Listener;
+import org.spongepowered.api.event.network.ClientConnectionEvent;
+import org.spongepowered.api.text.channel.MessageChannel;
+import org.spongepowered.api.text.channel.MutableMessageChannel;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.concurrent.TimeUnit;
+
+public class NoteListener extends ListenerBase {
+
+    @Inject private NoteHandler handler;
+    @Inject private NoteConfigAdapter nca;
+    @Inject private UserDataManager userDataManager;
+    private final String showOnLogin = PermissionRegistry.PERMISSIONS_PREFIX + "note.showonlogin";
+
+
+    /**
+     * At the time the player joins, check to see if the player has any notes,
+     * if he does send them to users with the permission nucleus.note.showonlogin
+     *
+     * @param event The event.
+     */
+    @Listener
+    public void onPlayerLogin(final ClientConnectionEvent.Join event) {
+        if (!nca.getNodeOrDefault().isShowOnLogin()) {
+            return;
+        }
+
+        Sponge.getScheduler().createTaskBuilder().async().delay(500, TimeUnit.MILLISECONDS).execute(() -> {
+            Player player = event.getTargetEntity();
+            List<NoteData> notes = handler.getNotes(player);
+            if (notes != null && !notes.isEmpty()) {
+                MutableMessageChannel messageChannel = MessageChannel.permission(showOnLogin).asMutable();
+                messageChannel.send(Util.getTextMessageWithFormat("note.login.notify", player.getName()));
+                int noteNumber = 1;
+                for (NoteData note : notes) {
+                    Optional<UserService> optUserService = userDataManager.get(note.getNoter());
+                    if (!optUserService.isPresent()) {
+                        messageChannel.send(Util.getTextMessageWithFormat("note.login.nonoter", String.valueOf(noteNumber), note.getNote()));
+                        continue;
+                    }
+                    UserService userService = optUserService.get();
+
+                    messageChannel.send(Util.getTextMessageWithFormat("note.login.note", String.valueOf(noteNumber), note.getNote(), userService.getUser().getName()));
+                    noteNumber++;
+                }
+            }
+        }).submit(plugin);
```

Comment:
It depends how many other things that appear on login really. One or two might be OK, but these are things that would appear each time staff log in.

Perhaps a compromise might be to display up to, say, the latest three notes from the past three days. If the notes are older, don't display them, but tell the staff member the user has x notes.